bower.json

```diff
@@ -77,7 +77,8 @@
     "datatables.net-dt": "^3.2.2",
     "underscore": "^1.10.2",
     "sql-formatter": "^1.1.2",
-    "angularjs-dropdown-multiselect": "2.0.0-beta.1"
+    "angularjs-dropdown-multiselect": "2.0.0-beta.1",
+    "datatables.net-rowgroup": "^1.1.2"
```

Comment:
can you elaborate why this library added as viz related should be placed in flair-visualisation?